frontend/src/utils/cardButtons.js

```diff
@@ -0,0 +1,158 @@
+import {
+    READY,
+    REVIEW_FEEDBACK,
+    COMPLETE,
+    BLOCKED,
+    IN_PROGRESS,
+    IN_REVIEW,
+    TEAM_PERMISSIONS,
+    MANAGE_CARDS,
+    REVIEW_CARDS,
+    TRUSTED_REVIEWER,
+  } from "../constants";
+  
+  export function getTeamPermissions({ authUser, viewedUser }) {
+    let result = {};
```

Comment:
Is it one results, or results that you store in this object?

Should be plural, not singular